cli/connect/interactive.go

```diff
@@ -0,0 +1,245 @@
+func handleLanguageOption(trimmed string, console *Console) {
+	newLang := strings.TrimPrefix(trimmed, setLanguagePrefix)
+	if lang, ok := ParseLanguage(newLang); ok {
+		if err := ChangeLanguage(console.conn, lang); err != nil {
+			log.Warnf("Failed to change language: %s", err)
+		} else {
+			console.language = lang
+			log.Infof("Selected language: %s", console.language.String())
+		}
+	} else {
+		log.Warnf("Unsupported language: %s", newLang)
+	}
+}
+
+// handleWidthOption handles width interactive option for table/ttable output formats.
+func handleWidthOption(trimmed string, console *Console, formatterOpts *formatter.Opts) {
+	var maxWidthStr string
+	if strings.HasPrefix(trimmed, strings.TrimSpace(setTableColWidthShortPrefix)) {
+		maxWidthStr = strings.TrimPrefix(trimmed, setTableColWidthShortPrefix)
+	} else if strings.HasPrefix(trimmed, strings.TrimSpace(setTableColWidthPrefix)) {
+		maxWidthStr = strings.TrimPrefix(trimmed, setTableColWidthPrefix)
+	} else {
+		panic("there is no pattern cases for get width value")
+	}
+	valCasted, err := strconv.ParseInt(maxWidthStr, 10, 64)
+	if err == nil && valCasted >= 0 {
+		formatterOpts.ColWidthMax = int(valCasted)
+		if formatterOpts.ColWidthMax > 0 {
+			log.Infof("Selected max width: %v", formatterOpts.ColWidthMax)
+		} else {
+			log.Info("Selected max width: disabled")
+		}
+	} else {
+		log.Errorf("Max width must be non-negative number, but you gave: %v", maxWidthStr)
+	}
+}
+
+// handlePseudoGraphicsOption handles pseudo graphics enable/disable interactive option
+// for table/ttable output formats.
+func handlePseudoGraphicsOption(trimmed string, console *Console,
+	formatterOpts *formatter.Opts) {
+	if console.outputFormat == formatter.TableFormat ||
+		console.outputFormat == formatter.TTableFormat {
+		if trimmed == setPseudoGraphicsEnable {
+			formatterOpts.NoGraphics = false
+			log.Info("Pseudo graphics is enabled now")
+		} else if trimmed == setPseudoGraphicsDisable {
+			formatterOpts.NoGraphics = true
+			log.Info("Pseudo graphics is disabled now")
+		} else {
+			panic("there is no pattern cases for handling pseudo graphics enabling/disabling")
+		}
+	} else {
+		log.Error("Pseudo graphics enabling/disabling " +
+			"only allowed for table and ttable output formats")
+	}
+}
+
+// handleFormatOption handles output interactive option.
+func handleFormatOption(trimmed string, executorEval *string,
+	console *Console, formatterOpts *formatter.Opts) {
+	var newFormat string
+	if strings.HasPrefix(trimmed, setFormatPrefix) {
+		newFormat = strings.TrimPrefix(trimmed, setFormatPrefix)
+	}
+	if strings.HasPrefix(trimmed, setFormatShortPrefix) {
+		if trimmed == setFormatShortPrefix {
+			if console.outputFormat == formatter.DefaultFormat {
+				console.outputFormat = formatter.YamlFormat
+				formatterOpts.TransposeMode = formatter.IsTTableFormat(console.outputFormat)
+			}
+			console.outputFormat = 1 + (console.outputFormat % 4)
```

Comment:
Please, change `4` here to a constant in the `formatter` module.
```suggestion
			console.outputFormat = 1 + (console.outputFormat % 4)
```

Reply:
Done

Reply:
If I add a new format, then it is necessary to change the constant in the different module. This is not good, the changes should be in one place - formatter module.